dev/source/docs/building-setup-linux.rst

```diff
@@ -57,6 +58,37 @@ Now you should be able to build with waf as described in `BUILD.md <https://gith
 .. youtube:: 4B8BVskH0vc
 
 
+ccache for faster builds
```

Comment:
this is already part of the install script, so we don't need to do it manually

Reply:
moved back under other dists